Web Inspector: Remove obsolete platform version-specific styles for Web Inspector UI

rcaliman-apple opened the pull request:
#### 380f61c9af891cf77a4f28ef0668d0b2b9ef3330
<pre>
Web Inspector: Remove obsolete platform version-specific styles for Web Inspector UI
<a href="https://bugs.webkit.org/show_bug.cgi?id=281910">https://bugs.webkit.org/show_bug.cgi?id=281910</a>
<a href="https://rdar.apple.com/138406941">rdar://138406941</a>

Reviewed by BJ Burg and Devin Rousso.

Remove style guards for macOS Big Sur and Monterey.
Make Monterey styles the default macOS platform styles.

* Source/WebInspectorUI/UserInterface/Base/Main.js:
* Source/WebInspectorUI/UserInterface/Views/Main.css:
(body.mac-platform #undocked-title-area):
(body:not(.mac-platform) #undocked-title-area):
(@media (prefers-color-scheme: dark) body.mac-platform #undocked-title-area):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) #undocked-title-area):
(@media (prefers-color-scheme: dark) body.mac-platform.window-inactive #undocked-title-area):
(@media (prefers-color-scheme: dark) body:not(.mac-platform).window-inactive #undocked-title-area):
(body:is(.mac-platform.monterey, .mac-platform.big-sur) #undocked-title-area): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) #undocked-title-area): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) #undocked-title-area): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) #undocked-title-area): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive #undocked-title-area): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur).window-inactive #undocked-title-area): Deleted.
* Source/WebInspectorUI/UserInterface/Views/TabBar.css:
(body.mac-platform .tab-bar):
(body:not(.mac-platform) .tab-bar):
(body.window-inactive .tab-bar):
(body.mac-platform .tab-bar &gt; .tabs &gt; .item):
(body:not(.mac-platform) .tab-bar &gt; .tabs &gt; .item):
(body.mac-platform .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected):
(body.mac-platform:not(.docked) .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected):
(body:not(.mac-platform) .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected):
(body.mac-platform .tab-bar &gt; .tabs:not(.animating) &gt; .item:not(.selected, .disabled):hover):
(body:not(.mac-platform) .tab-bar &gt; .tabs:not(.animating) &gt; .item:not(.selected, .disabled):hover):
(body.window-inactive .tab-bar &gt; .tabs &gt; .item):
(@media (prefers-color-scheme: dark) body.mac-platform .tab-bar):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) .tab-bar):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) .tab-bar &gt; .tabs &gt; .item):
(@media (prefers-color-scheme: dark) body.mac-platform .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected):
(@media (prefers-color-scheme: dark) body.mac-platform .tab-bar &gt; .tabs:not(.animating) &gt; .item:not(.selected, .disabled):hover):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) .tab-bar &gt; .tabs:not(.animating) &gt; .item:not(.selected, .disabled):hover):
(@media (prefers-color-scheme: dark) body.window-inactive .tab-bar):
(@media (prefers-color-scheme: dark) body.mac-platform.window-inactive .tab-bar &gt; .tabs &gt; .item):
(@media (prefers-color-scheme: dark) body:not(.mac-platform).window-inactive .tab-bar &gt; .tabs &gt; .item):
(@media (prefers-color-scheme: dark) body.mac-platform.window-inactive .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected):
(@media (prefers-color-scheme: dark) body:not(.mac-platform).window-inactive .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected):
(body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar,): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs &gt; .item): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs &gt; .item): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs:not(.animating) &gt; .item:not(.selected, .disabled):hover): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs:not(.animating) &gt; .item:not(.selected, .disabled):hover): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar &gt; .tabs &gt; .item,): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar,): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs &gt; .item): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs:not(.animating) &gt; .item:not(.selected, .disabled):hover): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar &gt; .tabs:not(.animating) &gt; .item:not(.selected, .disabled):hover): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar,): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar &gt; .tabs &gt; .item): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar &gt; .tabs &gt; .item): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar &gt; .tabs &gt; .item:not(.disabled).selected): Deleted.
* Source/WebInspectorUI/UserInterface/Views/TimelineOverview.css:
(body.mac-platform.legacy .timeline-overview &gt; .graphs-container): Deleted.
* Source/WebInspectorUI/UserInterface/Views/Variables.css:
(&amp;.mac-platform):
(&amp;:not(.docked)):
(&amp;:is(.mac-platform.monterey, .mac-platform.big-sur)): Deleted.
(&amp;:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked)): Deleted.
(&amp;:is(.mac-platform.catalina):not(.docked)): Deleted.

Canonical link: <a href="https://commits.webkit.org/285754@main">https://commits.webkit.org/285754@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d7c682f8c3daf2fc93ab979b67ea355843a79f49

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/73717 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/53146 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/26528 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/78033 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/24955 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/62279 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/931 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/57983 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/16357 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/76784 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/47/builds/48124 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/63428 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/38383 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/44869 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/64/builds/20915 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/23288 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/66477 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/63/builds/21263 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/79606 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/1034 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/496 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/66324 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/1176 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/63438 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/65603 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/16207 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/9468 "Passed tests") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/119/builds/7650 "EWS skipped this build as PR had skip-ews label when EWS attempted to process it.") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/998 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/1027 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/1014 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/1033 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rcaliman-apple commented on Source/WebInspectorUI/UserInterface/Views/TabBar.css:
```diff
@@ -41,21 +41,20 @@
     --tab-bar-background: linear-gradient(to bottom, hsl(0, 0%, 78%), hsl(0, 0%, 72%));
 }
 
-body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar,
-body:not(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar {
```
This confusing selector was most likely written to win a specificity race (the arguments inside `:is()` and `:not()` [contribute to overall selector specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/:is#difference_between_is_and_where)):

> The specificity of the :not() pseudo-class is replaced by the specificity of the most specific selector in its comma-separated argument of selectors

The race is between `body.mac-platform.monterey` and `body.window-inactive` and the former wins. 
This actually caused a subtle bug where the styles for the unfocused undocked Web Inspector title bar were overwritten.

After we simplify the style-guarded selectors down to `body.mac-platform` it's becomes even specificity with `body.window-inactive`. We rely on CSS order for the `.window-inactive` to apply.

**Before**
<img width="537" alt="undocked-area-inactive-before" src="https://github.com/user-attachments/assets/68e9a359-c052-41d6-86b4-f10246d8ffe9">

**After**
<img width="538" alt="undocked-area-inactive-after" src="https://github.com/user-attachments/assets/73faf814-a925-4a77-95d0-f63c69b03f59">

This fixes the issue and brings the unfocused undocked Web Inspector UI title bar in line with macOS design (from white to gray):

**Before**
<img width="1019" alt="light-inactive-before" src="https://github.com/user-attachments/assets/1646f48c-ba69-41f7-b451-a393eee61d79">

**After**
<img width="1019" alt="light-inactive-after" src="https://github.com/user-attachments/assets/7710ee0e-2e54-4719-a117-7ed446a20ad7">

rcaliman-apple commented on Source/WebInspectorUI/UserInterface/Views/TimelineOverview.css:
```diff
@@ -193,10 +193,6 @@ body[dir=rtl] .timeline-overview.frames > .timeline-ruler:not(.both-handles-clam
     left: 0;
 }
 
-body.mac-platform.legacy .timeline-overview > .graphs-container {
-    top: 22px;
-}
-
```
The `legacy` class name is not applied anywhere. This selector never matches.

dcrousso commented on Source/WebInspectorUI/UserInterface/Base/Main.js:
```diff
@@ -233,9 +233,6 @@ WI.contentLoaded = function()
     // Add platform style classes so the UI can be tweaked per-platform.
     document.body.classList.add(WI.Platform.name + "-platform");
-    if (WI.Platform.version.name)
-        document.body.classList.add(WI.Platform.version.name);
```
it may be worth keeping this if we plan to keep `InspectorFrontendHost.platformVersionName` around so that in the future if things are added to it then we don't have to re-add this code

dcrousso commented on Source/WebInspectorUI/UserInterface/Views/TabBar.css:
```diff
@@ -350,40 +347,39 @@ body.window-inactive .tab-bar > .tabs.animating.closing-tab > .item:not(.disable
 @media (prefers-color-scheme: dark) {
-    body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar,
-    body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar {
+    body.mac-platform .tab-bar {
```
shouldn't this just be `.tab-bar`?  the `:is` and `:not` are mutually exclusive meaning that this would've matched _anything_

rcaliman-apple replied:
The `:is()` and `:not()` were mutually exclusive, but they were contributing to being same-level specificity with their light mode counterpart CSS rules. This way they won based on CSS order when in dark mode.

That's also why `body.mac-platform .tab-bar` needs to stay here in the dark mode media query – to be the same specificity as `body.mac-platform .tab-bar` from line 44 for light mode.

dcrousso commented on Source/WebInspectorUI/UserInterface/Views/Variables.css:
```diff
@@ -424,14 +424,10 @@ body {
         }
     }
 
-    &:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
+    &:not(.docked) {
```
shouldn't this be `&.mac-platform:not(.docked)`?

rcaliman-apple replied:
This generic style does look redundant with the specific style for mac platforms defined in Main.js:

https://github.com/WebKit/WebKit/blob/1907a445147bf44d8a54bd25067bc62ecc06b63f/Source/WebInspectorUI/UserInterface/Base/Main.js#L2650-L2655

But I'm hesitant to make the rule more specific because it collapses `#undocked-title-area` to zero on non-Mac platforms and that causes overlap within the UI. 

Tested by removing `mac-platform` from the class list. I don't know for sure how Web Inspector looks like on a non-Mac platform. I'll ask in WebKit Slack .<img width="1075" alt="Screenshot 2024-10-27 at 14 36 50" src="https://github.com/user-attachments/assets/a105e03f-330d-4c70-aadc-cf4a2cea7e75">

dcrousso commented on Source/WebInspectorUI/UserInterface/Views/Variables.css:
```diff
@@ -424,14 +424,10 @@ body {
-        --undocked-title-area-height: calc(22px / var(--zoom-factor));
-    }
-
     &.mac-platform {
```
can we merge this with the above rule?

ditto anywhere else in this PR where we now have multiple declarations with the same selector

rcaliman-apple replied:
Yes, I can do that. I avoided doing it in the first place to ensure the PR was easy to understand that only selectors were changed, not other styles. But I can do the merge now.

rcaliman-apple committed:
Web Inspector: Remove obsolete platform version-specific styles for Web Inspector UI
https://bugs.webkit.org/show_bug.cgi?id=281910
rdar://138406941

Reviewed by BJ Burg and Devin Rousso.

Remove style guards for macOS Big Sur and Monterey.
Make Monterey styles the default macOS platform styles.

* Source/WebInspectorUI/UserInterface/Base/Main.js:
* Source/WebInspectorUI/UserInterface/Views/Main.css:
(body.mac-platform #undocked-title-area):
(body:not(.mac-platform) #undocked-title-area):
(@media (prefers-color-scheme: dark) body.mac-platform #undocked-title-area):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) #undocked-title-area):
(@media (prefers-color-scheme: dark) body.mac-platform.window-inactive #undocked-title-area):
(@media (prefers-color-scheme: dark) body:not(.mac-platform).window-inactive #undocked-title-area):
(body:is(.mac-platform.monterey, .mac-platform.big-sur) #undocked-title-area): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) #undocked-title-area): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) #undocked-title-area): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) #undocked-title-area): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive #undocked-title-area): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur).window-inactive #undocked-title-area): Deleted.
* Source/WebInspectorUI/UserInterface/Views/TabBar.css:
(body.mac-platform .tab-bar):
(body:not(.mac-platform) .tab-bar):
(body.window-inactive .tab-bar):
(body.mac-platform .tab-bar > .tabs > .item):
(body:not(.mac-platform) .tab-bar > .tabs > .item):
(body.mac-platform .tab-bar > .tabs > .item:not(.disabled).selected):
(body.mac-platform:not(.docked) .tab-bar > .tabs > .item:not(.disabled).selected):
(body:not(.mac-platform) .tab-bar > .tabs > .item:not(.disabled).selected):
(body.mac-platform .tab-bar > .tabs:not(.animating) > .item:not(.selected, .disabled):hover):
(body:not(.mac-platform) .tab-bar > .tabs:not(.animating) > .item:not(.selected, .disabled):hover):
(body.window-inactive .tab-bar > .tabs > .item):
(@media (prefers-color-scheme: dark) body.mac-platform .tab-bar):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) .tab-bar):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) .tab-bar > .tabs > .item):
(@media (prefers-color-scheme: dark) body.mac-platform .tab-bar > .tabs > .item:not(.disabled).selected):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) .tab-bar > .tabs > .item:not(.disabled).selected):
(@media (prefers-color-scheme: dark) body.mac-platform .tab-bar > .tabs:not(.animating) > .item:not(.selected, .disabled):hover):
(@media (prefers-color-scheme: dark) body:not(.mac-platform) .tab-bar > .tabs:not(.animating) > .item:not(.selected, .disabled):hover):
(@media (prefers-color-scheme: dark) body.window-inactive .tab-bar):
(@media (prefers-color-scheme: dark) body.mac-platform.window-inactive .tab-bar > .tabs > .item):
(@media (prefers-color-scheme: dark) body:not(.mac-platform).window-inactive .tab-bar > .tabs > .item):
(@media (prefers-color-scheme: dark) body.mac-platform.window-inactive .tab-bar > .tabs > .item:not(.disabled).selected):
(@media (prefers-color-scheme: dark) body:not(.mac-platform).window-inactive .tab-bar > .tabs > .item:not(.disabled).selected):
(body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar,): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs > .item): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs > .item): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs > .item:not(.disabled).selected): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) .tab-bar > .tabs > .item:not(.disabled).selected): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs > .item:not(.disabled).selected): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs:not(.animating) > .item:not(.selected, .disabled):hover): Deleted.
(body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs:not(.animating) > .item:not(.selected, .disabled):hover): Deleted.
(body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar > .tabs > .item,): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar,): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs > .item): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs > .item:not(.disabled).selected): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs > .item:not(.disabled).selected): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs:not(.animating) > .item:not(.selected, .disabled):hover): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur) .tab-bar > .tabs:not(.animating) > .item:not(.selected, .disabled):hover): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar,): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar > .tabs > .item): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar > .tabs > .item): Deleted.
(@media (prefers-color-scheme: dark) body:is(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar > .tabs > .item:not(.disabled).selected): Deleted.
(@media (prefers-color-scheme: dark) body:not(.mac-platform.monterey, .mac-platform.big-sur).window-inactive .tab-bar > .tabs > .item:not(.disabled).selected): Deleted.
* Source/WebInspectorUI/UserInterface/Views/TimelineOverview.css:
(body.mac-platform.legacy .timeline-overview > .graphs-container): Deleted.
* Source/WebInspectorUI/UserInterface/Views/Variables.css:
(&.mac-platform):
(&:not(.docked)):
(&:is(.mac-platform.monterey, .mac-platform.big-sur)): Deleted.
(&:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked)): Deleted.
(&:is(.mac-platform.catalina):not(.docked)): Deleted.

Canonical link: https://commits.webkit.org/285754@main
